### ISSUE TITLE
mysql: replace, insert, and update options

### ISSUE THREAD

**Issue by n0v1c3** [mikeboiko/tbl2mysql#5]
Add options/parameters to the script to allow for the option of doing a full replace, inserting all data into the current table, or updating existing while inserting new rows of data.

**Comment by mikeboiko** [comment]
Yes, I like this idea. I think it would be cool to add a feature to preserve the over-written data with the one of the standard methods that we already use for keeping historical records.
